WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -73,126 +78,110 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 ActivityLauncher.viewCurrentBlogComments(getActivity());
             }
         });
 
-        WPTextView themesTextView = (WPTextView) rootView.findViewById(R.id.my_site_themes_text_view);
-        themesTextView.setOnClickListener(new View.OnClickListener() {
+        mThemesContainer.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 ActivityLauncher.viewCurrentBlogThemes(getActivity());
             }
         });
 
-        WPTextView settingsTextView = (WPTextView) rootView.findViewById(R.id.my_site_settings_text_view);
-        settingsTextView.setOnClickListener(new View.OnClickListener() {
+        rootView.findViewById(R.id.row_settings).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 ActivityLauncher.viewBlogSettingsForResult(getActivity(), mBlog);
             }
         });
 
-        WPTextView viewAdminTextView = (WPTextView) rootView.findViewById(R.id.my_site_view_admin_text_view);
-        viewAdminTextView.setOnClickListener(new View.OnClickListener() {
+        rootView.findViewById(R.id.row_admin).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 ActivityLauncher.viewBlogAdmin(getActivity(), mBlog);
             }
         });
 
-        LinearLayout addPostContainer = (LinearLayout) rootView.findViewById(R.id.my_site_posts_add_button_container);
-        addPostContainer.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                ActivityLauncher.addNewBlogPostOrPage(getActivity(), mBlog, false);
-            }
-        });
-
-        LinearLayout addPageContainer = (LinearLayout) rootView.findViewById(R.id.my_site_pages_add_button_container);
-        addPageContainer.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                ActivityLauncher.addNewBlogPostOrPage(getActivity(), mBlog, true);
-            }
-        });
-
-        LinearLayout addMediaContainer = (LinearLayout) rootView.findViewById(R.id.my_site_media_add_button_container);
-        addMediaContainer.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                ActivityLauncher.addMedia(getActivity());
-            }
-        });
-
         refreshBlogDetails();
 
         return rootView;
     }
 
+    private void showSitePicker() {
+        if (isAdded()) {
+            ReaderAnim.showFab(mFabView, false);
+            int localBlogId = (mBlog != null ? mBlog.getLocalTableBlogId() : 0);
+            ActivityLauncher.showSitePickerForResult(getActivity(), localBlogId);
+        }
+    }
+
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
Yeah, it that rare situation I don't think the lack of opening animation
matters.
On May 25, 2015 9:56 AM, "Maxime Biais" <notifications@github.com> wrote:

> In
> WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java
> <https://github.com/wordpress-mobile/WordPress-Android/pull/2752#discussion_r30980921>
> :
>
> >          refreshBlogDetails();
> >
> >          return rootView;
> >      }
> >
> > +    private void showSitePicker() {
> > +        if (isAdded()) {
> > +            ReaderAnim.showFab(mFabView, false);
> > +            int localBlogId = (mBlog != null ? mBlog.getLocalTableBlogId() : 0);
> > +            ActivityLauncher.showSitePickerForResult(getActivity(), localBlogId);
> > +        }
> > +    }
> > +
> >      @Override
> >      public void onActivityResult(int requestCode, int resultCode, Intent data) {
>
> It's minor, but opening animation won't work in that case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wordpress-mobile/WordPress-Android/pull/2752/files#r30980921>
> .
>